src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor_code.go

```diff
@@ -91,13 +93,14 @@ func (a *StoragePowerActorCode_I) WithdrawBalance(rt Runtime, amount actor.Token
 
```

Comment:
In general, would be good if we can always have the code error out prominently (e.g., panic("TODO")) whenever something is still incomplete, rather than leaving the variable uninitialized. This also applies to line 61 `var msgValue actor.TokenAmount` and in several other places (I can't seem to add GitHub comments on lines that weren't changed in this diff, though).